CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## [0.28.0](https://github.com/redhat-developer/vscode-xml/milestone/39?closed=1) (March 5, 2025)
+
+### Breaking Changes
+ * Update to Java 11. See [eclipse-lemminx/lemminx#1721](https://github.com/eclipse-lemminx/lemminx/pull/1721)
+   * _Rationale_: Several of lemminx's dependencies now require Java 11, so we were unable to update to the latest versions until we adopted Java 11.
+
+### Bug Fixes
+
+ * Fix binary not working on older CPUs. See [#1055](https://github.com/redhat-developer/vscode-xml/issues/1055).
```

Comment:
Support LemMinX binary on older CPU architectures.

Reply:
They have the same architecture (x86_64) but don't support some of the instructions that were being used in the binary since those came from extensions on top of the base set of instructions.

Reply:
"Support LemMinx binary against older instruction sets" ?

Reply:
"Avoid using cpu instruction set extensions in order to support older CPUs when using the binary" ?

Reply:
I switched it over to what I wrote for lemminx. Does this make sense?